recovery/elliptics_recovery/types/merge.py

```diff
@@ -686,6 +683,98 @@ def succeeded(self):
         return self.result
 
 
+class ServerSendRecovery(object):
```

Comment:
Please add description of how does this class work, what is the logic behind it and so on, as much information as possible